flexget/plugins/filter/unique.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+
+from flexget import plugin
+from flexget.event import event
+from flexget.config_schema import one_or_more
+
+log = logging.getLogger('unique')
+
+
+class Unique(object):
+    """
+    Take action on entries with duplicate fields, except for the first item
+
+    Reject the second+ instance of every movie:
+
+      unique:
+        field:
+          - imdb_id
+          - movie_name
+        action: reject
+    """
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'field': one_or_more({'type': 'string'}),
+            'action': {'enum': ['accept', 'reject']},
+        },
+        'required': ['field'],
+        'additionalProperties': False
+    }
+
+    def prepare_config(self, config):
+        if isinstance(config, bool) or config is None:
+            config = {}
+        config.setdefault('action', 'reject')
+        if not isinstance(config['field'], list):
+            config['field'] = [config['field']]
+        return config
+
+    def extract_fields(self, entry, field_names):
+        return [ entry[field] for field in field_names ]
```

Comment:
Same here: https://github.com/flexget/Flexget/blob/master/flexget/plugins/filter/duplicates.py#L38
Think we should skip entries/prospects without the required fields instead?